examples/contrib/readhar.py

```diff
@@ -0,0 +1,134 @@
+"""Reads HAR files into flow objects"""
+import asyncio
+import codecs
+import json
+import logging
+import time
+
+from mitmproxy import command
+from mitmproxy import connection
+from mitmproxy import ctx
+from mitmproxy import exceptions
+from mitmproxy import http
+from mitmproxy import types
+
+
+class ReadHar:
+    def __init__(self):
+        self.harFile = {}
+        self.flows = []
+
+    def fix_headers(self, request_headers):
+        """Changes provided headers into flow-friendly format"""
+        flow_headers = []
+        for header in request_headers:
+            if type(header) == dict and header.get("name", None):
```

Comment:
Nit: `isinstance(header, dict)` is slightly better here. Two reasons:

1. `type(header) == dict` does not work for dict subclasses. That's not really relevant here at all, but you can avoid any problems in that regard by just consistently using `isinstance`.
2. The `header.get("name", None)` does not save things I believe. If there is no name attribute, `header[0]` in line 29 will also just crash as well.